### ISSUE TITLE
E0583: file not found for module

### ISSUE THREAD

**Issue by coolreader18** [llogiq/flamer#45]
When I put a `#![flame]` attribute inside of a module, it gives me an error "file not found for module `module_name`". It gives me the same error randomly when I apply it to the `mod` item in its parent module.

**Comment by coolreader18** [comment]
Somewhat related to this, what would you recommend to be the easiest way to mark every/many functions in a crate as `#[flame]`? I saw `apply_attr` mentioned, but that no longer seems to work, and I'm having problems like this one with marking modules as `#[flame]`.

**Comment by coolreader18** [comment]
Somewhat related to this, what would you recommend to be the easiest way to mark every/many functions in a crate/mod as `#[flame]`? I saw `apply_attr` mentioned, but that no longer seems to work, and I'm having problems like this one with marking modules as `#[flame]`.11

**Comment by llogiq** [comment]
I thought I had a test for this. Is the `mod` in the same file as the annotation (not that it should matter, but I'd like to reproduce it correctly)?

**Comment by coolreader18** [comment]
Yes, it is. I had to specify the full path when doing it from the outside to get it to work:

```rust
#[flame]
#[path = "/home/coolreader18/.../src/module_name.rs"]
mod module_name;
```

**Comment by llogiq** [comment]
That path appears to not be relative to the project. Why?

* If you leave out the `#[flame]` annotation, does that work?
* If you leave out the `#[path ..]` annotation, instead moving the module into `src/module_name.rs` in your project (assuming it isn't already there), does that work?

**Comment by coolreader18** [comment]
The thing is, it is already in the expected position. It does work without the `#[flame]` annotation, but once I put it there, it says it can't find the module and nothing works except a fully qualified `#[path = "/..."]`.

**Comment by llogiq** [comment]
Strange. Flamer does nothing out of the ordinary while handling mods. I don't have time right now to look into this. Can you come up with a minimal example to reproduce the problem?
